python/pyspark/tests/test_worker.py

```diff
@@ -144,6 +144,15 @@ def test_with_different_versions_of_python(self):
         finally:
             self.sc.pythonVer = version
 
+    def test_reuse_worker_of_parallelize_xrange(self):
+        def get_worker_pid(input_rdd):
+            return input_rdd.map(lambda x: os.getpid()).collect()
```

Comment:
Let's just call `.map(lambda x: os.getpid()).collect()` twice. It's oneliner anyway.

Reply:
Copy, done in 4868e82.